### PR TITLE
elifulkerson.tcping version 1

### PR DESCRIPTION
**Pull request by lil-Dangerz**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66857)

### CODE REVIEW

File: manifests/e/elifulkerson/tcping/1/elifulkerson.tcping.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: elifulkerson.tcping
+PackageVersion: 1
+Installers:
+- Architecture: x86
+  InstallerType: exe
```
Comment (Trenly) [inline manifests/e/elifulkerson/tcping/1/elifulkerson.tcping.installer.yaml:8]:
```suggestion
  InstallerType: portable
```

File: manifests/e/elifulkerson/tcping/1/elifulkerson.tcping.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: elifulkerson.tcping
+PackageVersion: 1
```
Comment (Trenly) [inline manifests/e/elifulkerson/tcping/1/elifulkerson.tcping.installer.yaml:5]:
```suggestion
PackageVersion: 0.39
```

File: manifests/e/elifulkerson/tcping/1/elifulkerson.tcping.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: elifulkerson.tcping
+PackageVersion: 1
```
Comment (Trenly) [inline manifests/e/elifulkerson/tcping/1/elifulkerson.tcping.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.39
```

File: manifests/e/elifulkerson/tcping/1/elifulkerson.tcping.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: elifulkerson.tcping
+PackageVersion: 1
```
Comment (Trenly) [inline manifests/e/elifulkerson/tcping/1/elifulkerson.tcping.yaml:5]:
```suggestion
PackageVersion: 0.39
```

**Review by Trenly — CHANGES_REQUESTED** [review]
This is blocked by microsoft/winget-cli#182. Until the 1.3 client is released as the latest version of the client, we cannot accept portable executables

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see https://github.com/microsoft/winget-pkgs/pull/66857#pullrequestreview-1044096109

**Comment by ItzLevvie** [comment]
Close with reason: Inactive PR;

**Pull request closed without merging**